lib/bundler/dsl.rb

```diff
@@ -490,7 +469,7 @@ def check_primary_source_safety(source_list)
     end
 
     def warn_deprecated_git_source(name, replacement, additional_message = nil)
-      # TODO: 2.0 remove deprecation
+      # TODO: 3.0 remove deprecation
```

Comment:
I think we should instead keep the sources, and enable the deprecations instead (`Bundler::SharedHelpers.major_deprecation 2`). I can do that in #6933, since that's what that PR is about.